src/analytics/polling/usePollingAnalytics.ts

```diff
@@ -2,8 +2,8 @@ import { addPageAction, Properties, Analytics } from "analytics/addPageAction";
 import { useGetUserQuery } from "analytics/useGetUserQuery";
```

Comment:
I'm not sure if I should rename this file to something different 🙂

Reply:
useActivityAnalytics?